lonnsmodell-produksjon.md

```diff
@@ -1,64 +1,34 @@
-| 0     | 500 000 kr                 | 7 %           | 605 000 kr |
-| 1     | 505 000 kr                 | 8 %           | 625 000 kr |
-| 2     | 510 000 kr                 | 9 %           | 645 000 kr |
-| 3     | 515 000 kr                 | 10 %          | 665 000 kr |
-| 4     | 520 000 kr                 | 11 %          | 685 000 kr |
-| 5     | 525 000 kr                 | 12 %          | 705 000 kr |
-| 6     | 530 000 kr                 | 13 %          | 725 000 kr |
-| 7     | 535 000 kr                 | 14 %          | 745 000 kr |
-| 8     | 540 000 kr                 | 15 %          | 765 000 kr |
-| ...   | ...                        | ...           | ...        |
-| N     | 500 000 kr + N \* 5 000 kr | 7 % + N \* 1% | -          |
-
-## Nivå 3
-
-Her vil det ligge et fåtall ansatte. Typisk må du være særskilt dyktig, veldig ettertraktet som utvikler, anerkjent
-foredragsholder eller på annet vis være av stor betydning for selskapet for å ligge her. Modellen har en fast årslønn, men vil øke provisjonen for hvert ledd. Fastlønnen er kr 600 000, og en starter med 15% provisjon. For hvert ledd øker provisjonsleddet med 1%.
-
-| Index | Fastlønn   | Provisjon      | Mållønn    |
-| ----- | ---------- | -------------- | ---------- |
-| 0     | 600 000 kr | 15 %           | 840 000 kr |
-| 1     | 600 000 kr | 16 %           | 856 000 kr |
-| 2     | 600 000 kr | 17 %           | 872 000 kr |
-| 3     | 600 000 kr | 18 %           | 888 000 kr |
-| ...   | ...        | ...            | ...        |
-| N     | 600 000 kr | 15 % + N \* 1% | -          |
+# Lønnsmodell for IT
+
+Lønn består av fastlønn og provisjon, og følger en fastsatt, trinnvis økning. Fastlønn øker med 5000kr og provisjon øker med 1% hvert år. Man går opp minst 1 trinn i året.
+
+Mållønn er etter alle praktiske formål den totale lønnen din. Denne er utgangspunktet for beregning av pensjon og skatt.
+
+Her følger et utsnitt av det overnevnte trinnvise oppbygningen:
+
+| Trinn | Fastlønn   | Provisjon | Mållønn    |
+| ----- | ---------- | --------- | ---------- |
+| 1     | 465 000 kr | 0 %       | 465 000 kr |
+| 2     | 470 000 kr | 1 %       | 488 860 kr |
+| 3     | 475 000 kr | 2 %       | 512 720 kr |
+| 4     | 480 000 kr | 3 %       | 536 580 kr |
+| 5     | 485 000 kr | 4 %       | 560 440 kr |
+| 6     | 490 000 kr | 5 %       | 584 300 kr |
+| 7     | 495 000 kr | 6 %       | 608 160 kr |
+| 8     | 500 000 kr | 7 %       | 632 020 kr |
+| 9     | 505 000 kr | 8 %       | 655 880 kr |
+| 10    | 510 000 kr | 9 %       | 679 740 kr |
+| 11    | 515 000 kr | 10 %      | 703 600 kr |
+| 12    | 520 000 kr | 11 %      | 727 460 kr |
+| 13    | 525 000 kr | 12 %      | 751 320 kr |
+| 14    | 530 000 kr | 13 %      | 775 180 kr |
+| 15    | 535 000 kr | 14 %      | 799 040 kr |
+| 16    | 540 000 kr | 15 %      | 822 900 kr |
+| 17    | 545 000 kr | 16 %      | 846 760 kr |
+| 18    | 550 000 kr | 17 %      | 870 620 kr |
```

Comment:
Bør vi nevne at maks provisjon er 17%?

Reply:
Tabellen er feil. Tallene vi bruker følger formelen 465 000 kr + N * 5 000 kr og N% - utgangspunktet for tallene er en omsetning på 1 500 000 kr.

Tabellen må begynne fra 0.